src/coreclr/jit/fginline.cpp

```diff
@@ -551,7 +551,7 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
         {
             return call->gtCallMethHnd;
         }
-        else
+        else if (call->IsGenericVirtual(m_compiler))
```

Comment:
This case is now more restrictive. Do we lose anything that we could get before?

Reply:
I don't think so. We bailed out all non-user calls before this change so previously any indirect call would not hit this path. 
https://github.com/dotnet/runtime/blob/bce6119e41ecfbcf630c369836770669604c22c6/src/coreclr/jit/fginline.cpp#L617